seed/views/v3/properties.py

```diff
@@ -199,12 +201,22 @@ def labels(self, request):
         Returns a list of all labels where the is_applied field
         in the response pertains to the labels applied to property_view
         """
-        labels = Label.objects.filter(
-            super_organization=self.get_parent_org(self.request)
```

Comment:
We should never be looking at a parent's org for labels because each org is responsible for their own labels.